doc/02-installation.md.d/For-Container.md

```diff
@@ -0,0 +1,179 @@
+	icinga/icinga2
+```
+
+This command will run the Icinga 2 agent container in detached mode, mounting the `/data` directory to a persistent
+volume named `icinga-agent`. As with the master container, you can adjust the volume name and other parameters as
+needed. The environment variables will be processed by the container's entrypoint script and perform a `icinga2 node setup`
+on your behalf, configuring the agent to connect to the master.
+
+You can repeat this step for each additional agent you want to add, ensuring that each agent has a unique hostname and
+zone name.
+
+## Icinga 2 API
+
+By default, if the `icinga2 node setup` command is run when starting the container, the Icinga 2 API will be enabled,
+and it will use a default API user named `root` with a randomly generated password. If you want to use your own API
+users and passwords, you can bind mount your api-users file from your host system into the
+`/data/etc/icinga2/conf.d/api-users.conf` in the container as described above.
+
+## Notifications
+
+By default, Icinga 2 does not send notifications when running in a containerized environment. However, it is possible
+to enable mail notifications by configuring [msmtp](https://wiki.archlinux.org/title/Msmtp) client in the container.
+The binary is already included in the official Icinga 2 container image, so it just needs to be configured. In order
+to do this, you can either mount the `/etc/msmtprc` file from your host system into the container or provide the
+necessary configuration for the `~/icinga/.msmtprc` file via the `MSMTPRC` environment variable.
+
+## Environment Variables
+
+Most of the environment variables are used as parameters for the `icinga2 node setup` command. If you set any of these
+variables, and the `node setup` has not been run yet, the entrypoint script will automatically run the command for you.
+
+The following environment variables are available:
+
+| Variable                                                 | Node setup CLI                                      |
+|----------------------------------------------------------|-----------------------------------------------------|
+| `ICINGA_ACCEPT_COMMANDS=1`                               | `--accept-commands`                                 |
+| `ICINGA_ACCEPT_CONFIG=1`                                 | `--accept-config`                                   |
+| `ICINGA_DISABLE_CONFD=1`                                 | `--disable-confd`                                   |
+| `ICINGA_MASTER=1`                                        | `--master`                                          |
+| `ICINGA_CN=icinga-master`                                | `--cn icinga-master`                                |
+| `ICINGA_ENDPOINT=icinga-master,2001:db8::192.0.2.9,5665` | `--endpoint icinga-master,2001:db8::192.0.2.9,5665` |
+| `ICINGA_GLOBAL_ZONES=global-config`                      | `--global_zones global-config`                      |
```

Comment:
The variable name is in the plural form, i.e. it suggests that multiple zones can be passed, but doesn't show how to pass multiple ones.

Reply:
Well it also didn't in the original README.md and it didn’t even support [multiple global zones](https://github.com/Icinga/docker-icinga2/blob/master/entrypoint/main.go#L79-L82) at all. I might just add the support for comma separated ones just like for the other options and expand them to `--global_zones one_zone --global_zones ...` in the entrypoint script.